docs/deployment/migration-guide.md

```diff
@@ -32,6 +32,8 @@
   relative to `$KYUUBI_HOME` instead of `$PWD`.
 * Since Kyuubi 1.8, PROMETHEUS is changed as the default metrics reporter. To restore previous behavior,
   set `kyuubi.metrics.reporters=JSON`.
+* Since Kyuubi 1.8.1, for `DELETE /batches/${batchId}`, `proxyUser` and `hive.server2.proxy.user`
```

Comment:
this should be listed in

```
## Upgrading from Kyuubi 1.8.0 to 1.8.1
```

but seems that change is not necessary to be mentioned